README.md

```diff
@@ -30,10 +31,46 @@ url: '[user].github.io/[repo]'
 ```
 
 There is no need to call this package directly, once `config.yaml` has been updated
-[{sandpaper}] will detect it and copy the styling and templates to your lesson website.
+[{sandpaper}] will detect it and copy the styling and templates to your lesson
+website when building in GitHub pages.
 
-Once `config.yaml`, typical guides from The Carpentries can be followed to deploy
-locally or to GitHub pages.
+Once `config.yaml` has been customised, typical guides from The Carpentries can be
+followed to deploy locally or to GitHub pages.
+
+### Applying Varnish locally
+
+When rendering the site locally the varnish will not, by default, be applied since
+it is not available. A few extra steps to setup up a [{renv}] and install the
+necessary packages are required.
+
+Initialise an `renv` in the workbench repository you have cloned.
+
+``` bash
+cd ~/path/to/workbench/repo/
+Rscript -e "renv::init()"
+```
+
+Start R and install this varnish and the [{sandpaper}] package (which will pull in
+all dependencies).
+
+``` r
+> renv::install("RSE-Sheffield/uos-varnish")
+> options(repos = c(
+    carpentries = "https://carpentries.r-universe.dev/",
+    CRAN = "https://cran.rstudio.com/"))
+> renv::install("sandpaper", dep = TRUE)
+> renv::snapshot()
+```
+
+You can now build and serve the pages with University of Sheffield varnish.
+
+``` r
+> library(uosvarnish)
```

Comment:
Ah, I was thinking something like this was missing whilst working through the guide!